Formula/crystal.rb

```diff
@@ -37,18 +36,16 @@ class Crystal < Formula
   depends_on "pcre"
 
   resource "boot" do
-    url "https://github.com/crystal-lang/crystal/releases/download/0.26.0/crystal-0.26.0-1-darwin-x86_64.tar.gz"
```

Comment:
@bcardiff should this be the 0.27 URL?

Reply:
No. It’s always the previous version of the compiler.

Reply:
@tritowntim This is because Crystal (0.27.0) is compiled with the previous Crystal version, which is 0.26.1